src/components/CommentTeaser/CommentTeaserHeader.js

```diff
@@ -11,17 +12,13 @@ const styles = {
   title: css({
     ...sansSerifMedium16,
     color: colors.text,
-    whiteSpace: 'nowrap',
-    overflow: 'hidden',
-    textOverflow: 'ellipsis'
+    ...ellipsize
   }),
   subtitle: css({
     ...sansSerifRegular14,
     color: colors.lightText,
     lineHeight: 1.1,
```

Comment:
That was my fix for descenders, we may wanna use your 20px lineHeight here as well for consistency.